Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -1125,38 +1131,72 @@ def fetch_incidents(
     incidents = list()
     next_fetch = from_time
     for record in records:
-        incident, incident_created_time = client.record_to_incident(
-            record, app_id, date_field, day_first=day_first, offset=offset
-        )
+        incident, incident_created_time = client.record_to_incident(record, app_id, fetch_param_id)
+        # Encountered that sometimes, somehow, on of next_fetch/incident_created_time are not UTC.
+        next_fetch = next_fetch.replace(tzinfo=timezone.utc)
+        incident_created_time = incident_created_time.replace(tzinfo=timezone.utc)
         if incident_created_time > next_fetch:
```

Comment:
lets add an `else` here which will print debug log the `incident_created_time` and the `next_fetch`